crates/turbopack-ecmascript/tests/analyzer/graph/conditional-import/input.js

```diff
@@ -1,5 +1,7 @@
 const a = import(true ? "a" : "b")
```

Comment:
This is testing your tenary support and should evaluate to a

Reply:
I wonder how this already works before

Reply:
Those tenary passes evaluation at

https://github.com/vercel/turbo/pull/6452/commits/3c2d29cb1cf6e897995b21997374617a6551fcc4#diff-2c7634641094d97670b756458b73ab7bc5f6fba2da97471427a6c95795b8562dR429, while dynamic tenary evaluation doesn't seem to pass, so it goes to jsvalue:alternatives.